pkg/util/log/log.go

```diff
@@ -0,0 +1,31 @@
+package log
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/dskit/dslog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/weaveworks/common/logging"
+	"github.com/weaveworks/common/server"
+)
+
+var (
+	// Logger is a shared go-kit logger.
+	// TODO: Change all components to take a non-global logger via their constructors.
+	// Prefer accepting a non-global logger as an argument.
+	Logger = log.NewNopLogger()
+)
```

Comment:
Where is this logger still used? Should we rather address the TODO?

Reply:
@jeschkies I hope to be able to return to this soon :/